pkg/virt-operator/creation/components/webhooks.go

```diff
@@ -429,7 +429,7 @@ const VirtApiServiceName = "virt-api"
 
 const VirtControllerServiceName = "virt-controller"
 
-const VirtHandlerServiceName = "virt-controller"
+const VirtHandlerServiceName = "virt-handler"
```

Comment:
whoa... I'm confused how this ever worked if we got the service name wrong.  Do you have a handle on what the implications of an issue like this?

Reply:
Nope, just randomly noticed that!